src/PokemonGymOwner.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class PokemonGymOwner {
```

Comment:
`name` en `pokemons` zitten ook in PokemonTrainer, dus je had voor deze klasse `extends PokemonTrainer` kunnen gebruiken. Daarmee voorkom je herhaling in je code.